spec/models/ext_management_system_spec.rb

```diff
@@ -413,8 +413,8 @@
     it "queues up destroy for child_managers" do
       child_manager = FactoryGirl.create(:ext_management_system)
       ems = FactoryGirl.create(:ext_management_system, :child_managers => [child_manager])
-      expect(described_class).to receive(:schedule_destroy_queue).with(ems.id)
-      expect(described_class).to receive(:schedule_destroy_queue).with(child_manager.id)
+      expect(described_class).to receive(:schedule_destroy_queue).with(ems.id, Integer)
+      expect(described_class).to receive(:schedule_destroy_queue).with(child_manager.id, Integer)
```

Comment:
I think you also need a test for rescheduling a destroy.

Reply:
👍 will do

Reply:
Done